pkg/query-service/app/http_handler.go

```diff
@@ -5489,13 +5504,83 @@ func (aH *APIHandler) getQueueOverview(w http.ResponseWriter, r *http.Request) {
 	aH.Respond(w, results)
 }
 
-func (aH *APIHandler) getCeleryOverview(w http.ResponseWriter, r *http.Request) {
+func (aH *APIHandler) getDomainList(w http.ResponseWriter, r *http.Request) {
+	thirdPartyQueryRequest, apiErr := ParseRequstBody(r)
+
+	if apiErr != nil {
+		zap.L().Error(apiErr.Err.Error())
+		RespondError(w, apiErr, nil)
+		return
+	}
+
+	queryRangeParams, err := thirdPartApi.BuildDomainList(thirdPartyQueryRequest)
+
+	if err := validateQueryRangeParamsV3(queryRangeParams); err != nil {
+		zap.L().Error(err.Error())
```

Comment:
don't this is error logging is required, since we are doing respondError it will take care of the error

Reply:
I thought of logging it so that the log is also present in query service, the respondError func will only throw the error as API response.
We so this in other handlers as well like `QueryRangeV4`

Reply:
but not all of these are requred, keep the very important ones, I will let you decide.